us.zoom.Zoom.json

```
@@ -11,10 +11,14 @@
         "--socket=pulseaudio",
         "--share=network",
         "--device=all",
-        "--filesystem=home"
+	"--filesystem=~/Documents/Zoom:create",
```

```suggestion
        "--filesystem=~/Documents/Zoom:create",
```

But really I already know it works, we have exactly same override there.

---

No really sure where you found that. Seeing in my repo, there's no problem: https://github.com/TheEvilSkeleton/us.zoom.Zoom/blob/beta/us.zoom.Zoom.json#L14

---

Weird. I wasn't able to see this issue on my end, Thank you